piecewise/regressor.py

```diff
@@ -220,7 +270,7 @@ def _preprocess(t, v):
     return t_arr, v_arr
 
 
-def _get_initial_segments(t, v):
+def _get_initial_segments_and_merges(t, v):
     """ Returns a list of Segments. Each Segment is of length 1, 2, or 3. They
```

Comment:
This docstring should be updated to reflect the new return type.

Reply:
Done